contracts/src/bridge/Outbox.sol

```diff
@@ -182,8 +182,12 @@ contract Outbox is DelegateCallAware, IOutbox {
         bytes32 calcRoot = calculateMerkleRoot(proof, index, item);
         if (roots[calcRoot] == bytes32(0)) revert UnknownRoot(calcRoot);
 
-        if (spent[index]) revert AlreadySpent(index);
-        spent[index] = true;
+        uint256 spentIndex = index / 256;
+        uint256 bitOffset = index - spentIndex * 256;
```

Comment:
Let's use 255 instead of 256 here.. this will leave the most significant bit reserved.
We might do something clever with this reserved bit later, to avoid 1/256 users paying more gas for the release.